00_introduction/22_architecture_root_server.txt

```diff
@@ -14,19 +14,32 @@ Operating System.
 Any CPU provided on commodity servers available nowaday should be fine. On
 a VM server, you should dedicate at least one core to the Rudder server.
 
-At least 1024 MB of RAM must be available on the server, depending on the
-base requirements of your operating system.
+The RAM needed by the server depends essentially of the number of nodes. 
 
-If you have a large number of nodes (more than 300), it is recommanded to
-give to Rudder web application more RAM (2GB) as explained in 
-the chapter [[jetty, about Jetty configuration]], and so to have at least
-3GB of RAM available on the server.  
+The base requirement is to have a server with 2GB of RAM. 
+For production environment, and as soon as you have more than 50 nodes, 
```

Comment:
why 50 ? I know 4 GB is not a lot, but reading this sentence, it makes me feel it is really gluton.
"On production environment, when managing more than XX nodes, it is recommended to dedicate at least 4GB of RAM to the server."